config/application.rb

```diff
@@ -343,6 +344,30 @@ def init_dynflow
     def setup_auditing
       Audit.send(:include, AuditSearch)
     end
+
+    def notify_deprecations
+      if ActiveRecord::Base.connection.adapter_name.downcase.starts_with?('mysql')
+        blueprint = NotificationBlueprint.find_by_name('feature_deprecation')
+        Foreman::Deprecation.deprecation_warning('1.24', "Support for MySQL is deprecated, migrate to PostgreSQL")
```

Comment:
there was a discussion to postpone this by one version, is it 1.24 now then?

Reply:
Yeah, changing to 1.25.